python/src/aac/__init__.py

```diff
@@ -12,7 +12,7 @@
 import logging
 import os
 
-__version__ = "0.1.10"
+__version__ = "0.1.11"
```

Comment:
Maybe that's worth an edit for the PR template, reminder about bumping appropriate package versions for changes?

Reply:
Hmmm that's a good question. It could be helpful to have a reminder, but every PR won't have a version bump since we have multiple PRs per release at the moment.

Reply:
Per conversation in scrum today we reached the agreement of putting a reminding comment of considering if a version bump is necessary within the PR template. And since it's a one line text update it would be okay to put in with this PR.